apps/framework-docs/src/pages/stack/workflows/schedule-workflow.mdx

```diff
@@ -0,0 +1,306 @@
+  config=WorkflowConfig(
+    starting_task=check_system_health_task,
+    schedule="@every 5m",  # Check every 5 minutes
+    retries=0,  # Don't retry monitoring checks
+    timeout="30s"
+  )
+)
+```
+</Python>
+
+## Monitoring Scheduled Workflows
+
+### Development Environment
+
+If your dev server is running, you should see logs in the terminal when your scheduled workflow is executed:
+
+```bash filename="Terminal" copy
+moose dev
+```
+
+```txt filename="Terminal"
+[2024-01-15 12:00:00] Scheduled workflow 'daily-data-processing' started
+[2024-01-15 12:00:01] Task 'extract' completed successfully
+[2024-01-15 12:00:15] Task 'transform' completed successfully
+[2024-01-15 12:00:30] Task 'load' completed successfully
+[2024-01-15 12:00:30] Workflow 'daily-data-processing' completed successfully
+```
+
+### Checking Workflow Status
+
+You can check the status of scheduled workflows using the CLI:
+
+```bash filename="Terminal" copy
+# List all workflow executions
+moose workflow list
+
+# Check specific workflow status
+moose workflow status daily-data-processing
+
+# Get detailed execution history
+moose workflow status daily-data-processing --verbose
+```
+
+### Temporal Dashboard
+
+Access the Temporal dashboard to view scheduled workflow executions:
+
+```bash filename="Terminal" copy
+# Open Temporal dashboard (typically at http://localhost:8080)
+open http://localhost:8080
+```
+
+The dashboard shows:
+- Scheduled workflow definitions
+- Execution history and timing
+- Success/failure rates
+- Retry attempts and errors
+
+## Best Practices for Scheduled Workflows
+
+### Timeout and Retry Configuration
+
+Configure appropriate timeouts and retries for scheduled workflows:
+
+<TypeScript>
+```typescript filename="app/robust-scheduled-workflow.ts" copy
+export const robustScheduledWorkflow = new Workflow("robust-scheduled", {
+  startingTask: mainTask,
+  schedule: "0 3 * * *", // Run at 3 AM daily
+  retries: 2, // Retry failed workflows
+  timeout: "4h", // Allow sufficient time
+});
+
+export const mainTask = new Task<void, void>("main", {
+  run: async () => {
+    // Long-running task logic
+  },
+  retries: 3, // Retry individual tasks
+  timeout: "1h", // Task-level timeout
+});
+```
+</TypeScript>
+
+<Python>
+```python filename="app/robust_scheduled_workflow.py" copy
+def run_main_task() -> None:
+    # Long-running task logic
+    pass
+
+main_task = Task[None, None](
+  name="main",
+  config=TaskConfig(
+    run=run_main_task,
+    retries=3,  # Retry individual tasks
+    timeout="1h"  # Task-level timeout
+  )
+)
+
+robust_scheduled_workflow = Workflow(
+  name="robust-scheduled",
+  config=WorkflowConfig(
+    starting_task=main_task,
+    schedule="0 3 * * *",  # Run at 3 AM daily
+    retries=2,  # Retry failed workflows
+    timeout="4h"  # Allow sufficient time
+  )
+)
+```
+</Python>
+
+## Troubleshooting Scheduled Workflows
+
+### Common Issues
+
+• **Timezone considerations**: Cron schedules use UTC by default
```

Comment:
This list is showing up as a paragraph instead of bulleted list. Maybe `-` instead of `•`?